ruby/ql/src/experimental/cwe-347/EmptyJWTSecret.qhelp

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Applications encoding a JSON Web Token (JWT) may be vulnerable when it's not verified or algorithm is <code>none</code>.
+</p>
+</overview>
+
+<recommendation>
+<p>
+Use non-empty nor <code>None</code> values while encoding JWT payloads.
```

Comment:
Nitpick, I found this line a little bit unclear. Maybe something like:
```suggestion
JSON Web Tokens should be signed using a strong cryptographic algorithm and non-empty secret.
```
I'm open to other suggestions!

Reply:
👍👍👍